src/cli/cli_mdns.cpp

```diff
@@ -879,6 +879,64 @@ void Mdns::HandleIp4AddressResult(otInstance *aInstance, const otMdnsAddressResu
     Interpreter::GetInterpreter().mMdns.HandleAddressResult(*aResult, kIp4Address);
 }
 
+template <> otError Mdns::Process<Cmd("recordquerier")>(Arg aArgs[])
+{
+    // mdns recordquerier start|stop <record-type> <first-label> [<next-labels>]
```

Comment:
Update the CLI README as well?

Reply:
None of the CLI `mdns` commands are currently documented (they are only intended for testing). This has bee intentional, since we do not (yet) want to make them available for external use (other than Thread internal use-case). Later, we may change this and add documentation for them all.